gulpfile.js

```diff
@@ -30,14 +29,13 @@ const build = (lib, opts) =>
       .pipe(gulpif(argv.sourcemaps, sourcemaps.write('.', {sourceRoot: '../src'})))
       .pipe(gulp.dest(lib));
 
-gulp.task('default', ['build']);
-
 gulp.task('build', () =>
   build('lib', babelRc.env[majorVer >= 5 ? 'node5' : 'pre-node5'])
 );
 
-gulp.task('watch', ['build'], () => {
-  watch('src/**/*', () => {
-    gulp.start('build');
```

Comment:
I'm not a `gulp` expert by any means, but it seems like we're doing `gulp.task('build')` a lot in here. Is there a good way to clean that up a bit so it's more terse? (e.g. `const buildTask = gulp.task('build')`?)

Reply:
I don't think it matters whether tasks are exported using `gulp.task` or by exporting them from the module, it's just a matter of style.

As for the repeated use of `gulp.task('build')`, it looks like the best way to avoid that is to just use the function reference instead. i.e. `gulp.task('default', build)`

That can't be done as things are currently, because `build` is a helper function. The actual `build` task is an anonymous function that calls the `build` function. I see no reason not to merge those two functions into one. After that, instances of `gulp.task('build')` could be replaced with `build`.